.github/workflows/nodejs.yml

```
@@ -0,0 +1,28 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on: [ push ]
```

```suggestion
on: [ push, pull_request ]
```

Otherwise it would not build for branches pushed on other repos.